examples/chords.js

```diff
@@ -13,7 +17,7 @@ scribble.midi(clip);
 
 const clip2 = scribble.clip({
   notes: scribble.getChordsByProgression('C4 major', 'vi IV V I'),
-  pattern: 'x',
+  pattern: 'xxxx',
```

Comment:
Without changing this, it only had the `vi` chord in the output. I tested on master before my changes as well, and it had the same unwanted behavior.

Reply:
Yes the notes repeat if the pattern is longer but the other way round is not yet implemented.